### PR TITLE
Reservation intervals

### PR DESCRIPTION
**Pull request by ranta**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Add new reservation interval choices for hours 2 - 7

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of what's the fastest way to test your changes."

- Automated tests
- See Jira ticket

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)"

- [TILA-3177](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3177)

[TILA-3177]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/graphql/tests/test_reservations/test_reservation_create.py
Comment (matti-lamppu) [inline api/graphql/tests/test_reservations/test_reservation_create.py:505]: If you have the time, these should be rewritten as parametrized functional tests.

**Review by matti-lamppu — APPROVED** [review]

File: api/graphql/tests/test_reservations/test_reservation_create.py
Comment (ranta) [reply api/graphql/tests/test_reservations/test_reservation_create.py:505]: I will wait until #1115 is merged and then I can rewrite all of the tests in this file without worry of creating conflicts for you :)

**Pull request merged**
